Classes/OpenidService.php

```diff
@@ -518,7 +521,7 @@ protected function normalizeOpenID($openIDIdentifier)
      */
     protected function getBackPath()
     {
-        $extPath = ExtensionManagementUtility::siteRelPath('openid');
+        $extPath = ExtensionManagementUtility::extPath('openid');
```

Comment:
Did you test this?
extPath returns an absolute path, so that will most probably not work out

Reply:
> `ExtensionManagementUtility::siteRelPath('openid')` will return `typo3conf/ext/openid/lib/php-openid`

ehm what? what's going on here?

Reply:
Append after line 524 to get the same as from `siteRelPath`:
```php
$extPath = substr($path, strlen(Environment::getPublicPath() . '/'));
```